homepage/de.js

```diff
@@ -0,0 +1,74 @@
+{
+  "Integrations": "Integrationen",
+  "Pricing": "Preise",
+  "Apps": "Apps",
+  "Existing user login": "Bestehende Benutzer-Anmeldung",
+  "login": "Anmelden",
+  "Chat, for GitHub.": "Chat, für GitHub.",
+  "Get unlimited public rooms and one-to-one chats, free forever. Private team plans start from $5 per user per month. No credit card required.": "Unbegrenzt öffentliche Chat-Räume und Direkt-Chats, dauerhaft kostenfrei. Angebote für private Teams beginnen bei $5 USD pro Benutzer und Monat. Es wird keine Kredikarte vorausgesetzt.",
+  "Sign up free with Github": "Kostenfrei anmelden per Github",
+  "Sign up with Github": "Anmelden per Github",
+  "By signing up you agree to our <a %s>Terms and Conditions</a>": "Durch die Anmmeldung stimmst Du unseren <a %s>allgemeinen Geschäftsbedingungen</a> zu",
+  "Markdown power": "Markdown Unterstützung",
+  "Ever wanted full markdown support in chat? How about syntax highlighting?": "Bislang erfolglos nach einem Chat-System mit markdown Unterstützung gesucht? Wie steht es um Syntax-Highlighting?",
+  "Yip, we got your back.": "Yep, da haben wir es.",
+  "Private conversations": "Private Unterhaltungen",
+  "Public rooms are awesome (and free), but we get that some stuff is private. So we've got <a %s>plans for that</a>.": "Öffentliche Chat-Räume sind ausgezeichnet (und kostenfrei) – Du brauchen aber auch etwas für den internen Austausch? Dafür gibt es <a %s>diese Angebote</a>.",
+  "Developer friendly": "Entwicklerfreundlich",
+  "This is a picture of a cloud against a background of clouds. OMG! We <a %s>integrate</a> with a bunch of stuff in the cloud.": "Dies ist das Bild einer Wolke vor einem Hintergrund aus Wolken. OMG! Wir <a %s>verbinden</a> uns mit vielen Diensten in der Cloud.",
+  "Some of our communities": "Einige aus unseren Gruppen",
+  "Create your own": "Erstell Deine",
```

Comment:
"Eigenen Chatroom erstellen" like in [#13](https://github.com/gitterHQ/gitter-translations/pull/13/files#diff-02133d9a97f86dc218a62446d3e99843R20)